src/custom.json

```diff
@@ -0,0 +1,16 @@
+{
+    "title": {
+      "hide": false
```

Comment:
Make sure `"hide": false` is optional. By default fields should be shown.

Reply:
yes , default behaviour is to show on the grid for both these cases  1)the column is not present in the custom.json 2)the "hide" key is not present for the column